test/re/frida/MethodTest.java

```diff
@@ -118,3 +174,17 @@ public int returnZero() {
         return 0;
     }
 }
+
+class Collider {
+    static int Particle = 1;
+    
+    int Particle2 = 2;
+
+    int Particle() {
+        return 3;
+    }
+
+    static int Particle2(){
+        return 4;
+    }
```

Comment:
Field- and method-names should follow the coding style, i.e. `camelCase`, not `CamelCase`.